zookeeper-server/src/main/java/org/apache/zookeeper/server/watch/WatchManagerFactory.java

```diff
@@ -32,14 +32,14 @@
 
     public static final String ZOOKEEPER_WATCH_MANAGER_NAME = "zookeeper.watchManagerName";
 
-    public static IWatchManager createWatchManager() throws IOException {
+    public static IWatchManager createWatchManager(String watchers) throws IOException {
         String watchManagerName = System.getProperty(ZOOKEEPER_WATCH_MANAGER_NAME);
```

Comment:
`watchName` is better?

Reply:
Yeah, I will change the variable name @maoling.